### PR TITLE
Remove appearance attribute when changing mug from a label

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?209488

cc: @sravfeyn

### CODE REVIEW

File: tests/questionTypes.js [hunk 1]
```diff
@@ -429,8 +437,39 @@ require([
                     assert(ok, "Error not raised when changing " + from + " to " + to);
                     mug = util.getMug(mug.p.nodeID);
                     assert.equal(mug.__className, from.replace(" + Choices", ""));
+                    tearDown(from, to);
                 });
             });
+
+            it("should remove appearance attribute from label", function () {
+                    var from = "Trigger",
```
Comment (millerdev) [inline tests/questionTypes.js]: funny indentation (extra level?)

Comment (emord) [inline tests/questionTypes.js]: oops https://github.com/dimagi/Vellum/commit/9cca5d46d9bd275bc40de10fa47192ed20bf5fa7

**Comment by millerdev** [comment]
g2m on test pass

**Comment by czue** [comment]
brings me back to know that :coffee: is still a thing

**Pull request merged**